secant/Features/Settings/SettingsView.swift

```diff
@@ -41,6 +51,65 @@ struct SettingsView: View {
     }
 }
 
+extension SettingsView {
+    private func exportAndShareLogs() {
+        let tempDir = FileManager.default.temporaryDirectory
+        
+        // SDK logs
+        let sdkFileName = "sdkLogs.txt"
+        let tempSDKDir = tempDir.appendingPathComponent(sdkFileName)
+        
+        do {
+            let sdkLogsStr = LogStore.exportCategory(LoggerConstants.sdkLogs)
+            var result = ""
+            sdkLogsStr?.forEach({ line in
+                result.append(line)
+                result.append("\n\n")
+            })
+            
+            try result.write(to: tempSDKDir, atomically: true, encoding: String.Encoding.utf8)
+        } catch { }
```

Comment:
we should mark all these empty catch clauses with TODOs

Reply:
resolved